### PR TITLE
Replacing fruits with apples

### PR DESCRIPTION
**Pull request by ReggieTe**
**What does this Pr do ?**

Replacing fruit with apples as indicated with the output code

### CODE REVIEW

File: doc/filters/replace.rst [hunk 1]
```diff
@@ -6,12 +6,12 @@ format is free-form):
 
 .. code-block:: twig
 
-    {{ "I like %this% and %that%."|replace({'%this%': fruit, '%that%': "oranges"}) }}
+    {{ "I like %this% and %that%."|replace({'%this%': apples, '%that%': "oranges"}) }}
```
Comment (lyrixx) [inline doc/filters/replace.rst:9]:
```suggestion
    {{ "I like %fruit% and %that%."|replace({'%fruit%': apples, '%that%': "oranges"}) }}
```

File: doc/filters/replace.rst [hunk 2]
```diff
@@ -6,12 +6,12 @@ format is free-form):
...
     {# if the "fruit" variable is set to "apples", #}
     {# it outputs "I like apples and oranges" #}
 
     {# using % as a delimiter is purely conventional and optional #}
-    {{ "I like this and --that--."|replace({'this': fruit, '--that--': "oranges"}) }}
+    {{ "I like this and --that--."|replace({'this': apples, '--that--': "oranges"}) }}
```
Comment (lyrixx) [inline doc/filters/replace.rst:14]:
```suggestion
    {{ "I like fruit and --that--."|replace({'fruit': apples, '--that--': "oranges"}) }}
```

File: doc/filters/replace.rst (hunk 1 above)
Comment (xabbuh) [inline doc/filters/replace.rst:9]: Replacing `fruit` with `apples` here is wrong. As the comment below indicates the example assumes that there is a variable called `fruit` whose value is `apples`.

**Comment by fabpot** [comment]
Closing as the current doc is correct. I've used `fruit` as a variable that can contain any fruits, vs `oranges` that is a string.

**Pull request closed without merging**